source-reminder/Makefile

```
@@ -9,7 +9,22 @@ deploy: install-deps
 install-deps:
 	pip3 install -t $$(pwd) -r $$(pwd)/requirements.txt
 
-# builds to .aws-sam directory cleaner than "install-deps"
-# todo: switch deploy from manually zipping the directory to using AWS SAM
-sam-build:
-	sam build --use-container
\ No newline at end of file
+
+# docker commands
+build-image:
+	docker build -t source-reminder/lambda:local .
+
+run-container: build-image
+	docker run \
+		--rm -it \
+		-p 9000:8080 \
+		-v $$(pwd)/credentials.json:/var/task/credentials.json \
+		-v $$(pwd)/src/lambda_function.py:/var/task/lambda_function.py \
```

Just a small suggestion here, it wasn't running for me like the donations repo and adding these fixed it (learned from your thread in the summer). As well as quotes on line 10 because of `pwd`.

```suggestion
		-v "$$(pwd)/credentials.json:/var/task/credentials.json" \
		-v "$$(pwd)/src/lambda_function.py:/var/task/lambda_function.py" \
```

---

Oh great catch and suggestions here, @jbarrueta - thank you for noticing and adding!  Committed both your suggestions.